src/navigation/appNavigation.js

```diff
@@ -329,7 +335,6 @@ const walletConnectFlow = createStackNavigator(
 // HOME FLOW
 const homeFlow = createStackNavigator({
   [HOME]: HomeScreen,
-  [SETTINGS]: SettingsScreen,
```

Comment:
`[RECOVERY_SETTINGS]: RecoverySettingsScreen` is needed here so user could get back to home screen after pressing on back up toast.